src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -7,8 +13,21 @@ namespace Dynamo.PackageDependency
     /// </summary>
     public partial class PackageDependencyView : UserControl
     {
-        public PackageDependencyView()
+
+        internal IEnumerable<Graph.Workspaces.PackageDependencyInfo> Packages;
+
+        private DynamoModel dynamoModel;
+
+        public void WorkspaceOpened(WorkspaceModel obj)
+        {
+            Packages = obj.PackageDependencies;
+        }
+
+        public PackageDependencyView(DynamoViewModel dynamoViewModel)
```

Comment:
why do you need the DynamoViewModel? If you can avoid using it we can avoid exposing quite a big API surface.